distros/openshift/security-context-constraints/scc-config-management-monitoring-default.yaml

```diff
@@ -0,0 +1,21 @@
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+allowPrivilegeEscalation: false
+readOnlyRootFilesystem: true
+metadata:
+  name: scc-config-management-monitoring-default
+fsGroup:
+  type: MustRunAs
+  ranges:
+    - min: 2000
+      max: 2000
+groups:
```

Comment:
@dkosteck Just a question why aren't you Dropping the capabilities here too? Like you did above `requiredDropCapabilities`

Reply:
@vjayaramrh Did we specifically look into this field, or was this not gleaned from the trials on OCP therefore it wasn't added?

Reply:
We look at the KRM manifests to determine what capabilities needs to be added or dropped. In this case the

Reply:
Okay, so are you adding in all of them? I suppose you can continue this good practice.

Reply:
FYI Not dropping all capabilities does not mean all capabilities would become available. Only capabilities that are specifically added will be available.

Reply:
For OCP, there would need to be SCC's created to match the capabilities requested.

Reply:
Yes true, but still the process gets some basic capabilities even if we do not add capabilities. We can check that for each process via `capsh` command. 

But this is more about fine tunning `scc`. We can take this up in another MR.

Reply:
@arora-sagar Great points, additionally, we may want to look at firming up the securityContext sections of the pods/containers in the KRM files as well.